src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -317,18 +335,35 @@ export const VaultDetailPanels = ({
         </VaultActions>
       </Row>
 
-      {chartData && (
+      {chartData?.underlying && chartValue?.underlying && chartValue?.underlying !== '0' && (
```

Comment:
I think we should create a const in this cases to keep the html clean, like
`chartDataVisible = chartData?.underlying && chartValue?.underlying && chartValue?.underlying !== '0'` 

and then just do `{charchartDataVisible tData && (....`